inference-engine/src/transformations/include/ngraph_ops/type_relaxed.hpp

```diff
@@ -41,7 +41,8 @@ class TRANSFORMATIONS_API TypeRelaxedBase {
     /// the same index i.
     const element::Type& get_overridden_output_type(size_t outputIndex = 0) const {
         if (outputIndex >= m_output_data_types.size()) {
-            return element::undefined;
+            static element::Type undef_elem(element::undefined);
```

Comment:
I think you can avoid this static by changing the return type of this method. When you return by value, not by reference this should do the trick and you can go back to `return element::undefined`.

Reply:
Agree